### ISSUE TITLE
[Datepicker] initialMonth should be default from value/defaultValue

### ISSUE THREAD

**Issue by nghiepit** [mantinedev/mantine#485]
Hey,
Currently, the initial month of Datepicker is the current date if without specifying a date.

```tsx
   <DatePicker
        allowFreeInput
        placeholder="Select a date"
        value={new Date("2021-11-01")}
      />
```

**Comment by rtivital** [comment]
You should be able to set initial month with `initialMonth` prop, isn't it working for you?

**Comment by nghiepit** [comment]
`initialMonth`prop works.
But if not set `initialMonth` then the initial month should be set default is `value` prop instead of current date.

**Comment by rtivital** [comment]
Ah, I see, it's the issue with free input date picker only, will fix that

**Comment by rtivital** [comment]
Fixed in 3.3.5
